src/components/EmailSubscription/index.view.tsx

```diff
@@ -1,9 +1,67 @@
-import React from "react";
+import React, { useState } from "react";
+import axios, { AxiosResponse } from "axios";
+import { ReactComponent as Arrow } from "images/arrow.svg";
+
 import "./EmailSubscription.scss";
 
 const EmailSubscriptionForm: React.FC = () => {
+  const [userEmail, setUserEmail] = useState("");
+  const [feedbackMessage, setFeedbackMessage] = useState("");
+  const [showFeedback, setShowFeedback] = useState(false);
+
+  const handleInput = (event: React.FormEvent<HTMLInputElement>) => {
+    setUserEmail(event.currentTarget.value.trim());
+  };
+
+  const handleKey = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === "Enter") {
+      submitEmail();
+    }
+  };
+
+  const submitEmail = () => {
+    axios
+      .post(
+        "https://us-central1-cruzhacks-4a899.cloudfunctions.net/subscribe",
```

Comment:
Let's try to contain implementation constants (like API endpoints) as environment secrets when possible

Reply:
Can you clarify again how this is done or point me toward resources that help?

Reply:
Working on this. Will get back to you ASAP!